<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="3">
     <widget class="QMdiArea" name="myMDI"/>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(64, 191, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="save"/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_6">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/pencil.png</normaloff>:/icons/pencil.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/lastic.png</normaloff>:/icons/lastic.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRectangle">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/rectangle.png</normaloff>:/icons/rectangle.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLine">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/line.png</normaloff>:/icons/line.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_10">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/curve.png</normaloff>:/icons/curve.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEllipse">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="paintResource.qrc">
          <normaloff>:/icons/ellipse.png</normaloff>:/icons/ellipse.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>870</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_somethig">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionPlace_a_window"/>
    <addaction name="OpenFile"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuDraw">
    <property name="title">
     <string>Pen</string>
    </property>
    <addaction name="actionPen_Color"/>
    <addaction name="actionPen_Width"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionTile"/>
    <addaction name="actionCascade"/>
   </widget>
   <addaction name="menuAdd_somethig"/>
   <addaction name="menuDraw"/>
   <addaction name="menuEdit"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPlace_a_window">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="OpenFile">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionLine">
   <property name="text">
    <string>line</string>
   </property>
  </action>
  <action name="actionRect">
   <property name="text">
    <string>rectangle</string>
   </property>
  </action>
  <action name="actionTile">
   <property name="text">
    <string>Tile</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Cascade</string>
   </property>
  </action>
  <action name="actionPen_Color">
   <property name="text">
    <string>Pen Color...</string>
   </property>
  </action>
  <action name="actionPen_Width">
   <property name="text">
    <string>Pen Width...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="paintResource.qrc"/>
 </resources>
 <connections/>
</ui>
